src/routes/blog/_query.ts

```diff
@@ -103,6 +113,14 @@ export async function fetchNewsletters(): Promise<Newsletter[]> {
     body: JSON.stringify({ query: newslettersQuery }),
   });
 
-  const newsletters = formatNewsletters(await response.json());
-  return newsletters;
+  let newsletters = formatNewsletters(await response.json());
```

Comment:
Since we are expecting to return a property `posts`, let's name this variable `posts` for convenience.
```suggestion
  const posts = formatNewsletters(await response.json());
```

Reply:
Also it's good practice to use `const` whenever u know for a fact the variable will never be reassigned

Reply:
I used `let` because it will be reassigned at line 120.

Reply:
oop- yea i just noticed LOL I didnt read that far down yet

Reply:
Good catch! Sounds good.